### ISSUE TITLE
ERR_OUT_OF_RANGE with prefixOffset = 1

### ISSUE THREAD

**Issue by staltz** [ssb-ngi-pointer/jitdb#114]
```
RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 0. Received 1
    at boundsError (internal/buffer.js:81:9)
    at Buffer.readUInt32LE (internal/buffer.js:215:5)
    at safeReadUint32 (/data/data/se.manyver/files/nodejs-project/index.js:33305:21)
    at updatePrefixMapIndex (/data/data/se.manyver/files/nodejs-project/index.js:33324:26)
    at /data/data/se.manyver/files/nodejs-project/index.js:33552:13
    at Array.forEach (<anonymous>)
    at write (/data/data/se.manyver/files/nodejs-project/index.js:33550:27)
    at Object.o.write (/data/data/se.manyver/files/nodejs-project/index.js:16057:11)
    at Stream.Ixt5Co4taXzqE+NfnRNMXeuKkHgmTv864lAVqLsGMWA=.Stream._writeToSink (/data/data/se.manyver/files/nodejs-project/index.js:30899:33)
    at Stream.Ixt5Co4taXzqE+NfnRNMXeuKkHgmTv864lAVqLsGMWA=.Stream._handleBlock (/data/data/se.manyver/files/nodejs-project/index.js:30932:12) {
    code: \'ERR_OUT_OF_RANGE\'
}
```

**Comment by arj03** [comment]
So I thought I would just fix this and create a PR before going to bed, turns out I forgot to create a branch first :-( Anyway the problem is that [readUInt32LE](https://nodejs.org/api/buffer.html#buffer_buf_readuint32le_offset) only allows offsets Must satisfy 0 <= offset <= buf.length - 4. So for these smaller lengths we can't do prefixOffset.

**Comment by staltz** [comment]
Solved by #115
